st2debug/tests/integration/test_submit_debug_info.py

```diff
@@ -204,22 +197,18 @@ def test_encrypt_archive(self):
                           extract_path='/tmp')
 
     def test_encrypt_archive_with_custom_gpg_key(self):
```

Comment:
In a future PR it would also be great if you can add another test case which tests more things when providing a custom config - verify command name is script, verify command stuff is included in the output archive, etc.